### PR TITLE
Create error pages

### PR DESCRIPTION
**Pull request by clem21**
| Q               | A
| --------------- | -----
| Branch?         | 1.8 
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | mentioned in #12247 
| License         | MIT

### CODE REVIEW

File: app/Resources/TwigBundle/views/Exception/error403.html.twig [hunk 1]
```diff
@@ -1 +1 @@
-{% include '@SyliusShop/error403.html.twig' %}
+{#{% include '@SyliusShop/error403.html.twig' %}#}
```
Comment (lchrusciel) [inline app/Resources/TwigBundle/views/Exception/error403.html.twig:1]: We need to use it this way:
```suggestion
{% include ['@SyliusShop/error403.html.twig', @SyliusUi/error.html.twig] %}
```
Therefore, we will serve `error403` if it exists, and a generic error otherwise

File: src/Sylius/Bundle/ShopBundle/Resources/views/error403.html.twig [hunk 2]
```diff
@@ -1,10 +1,2 @@
...
-{% block content %}
-    <h2 class="ui center aligned icon header">
-        <img src="{{ asset('assets/shop/img/logo.png') }}" alt="Sylius logo" class="ui small image" />
-        <br/><br/>
-        {{ 'sylius.ui.the_page_you_are_looking_for_is_forbidden'|trans }}
-    </h2>
-
-{% endblock %}
+{% set message = 'sylius.ui.the_page_you_are_looking_for_is_forbidden'|trans %}
+{% include '@SyliusUi/error.html.twig' with {'status_code': '403', 'status_text': message}%}
```
Comment (lchrusciel) [inline src/Sylius/Bundle/ShopBundle/Resources/views/error403.html.twig:2]: Can you revert these changes? Previously created errors should stay as they were

File: templates/bundles/TwigBundle/Exception/error403.html.twig [hunk 3]
```diff
@@ -0,0 +1 @@
+{% include '@SyliusShop/error403.html.twig' %}
```
Comment (lchrusciel) [inline templates/bundles/TwigBundle/Exception/error403.html.twig:1]: We need to use it this way:
```suggestion
{% include ['@SyliusShop/error403.html.twig', @SyliusUi/error.html.twig] %}
```
Therefore, we will serve `error403` if it exists, and a generic error otherwise. Also, can you do the same for other errors in this folder?

File: app/Resources/TwigBundle/views/Exception/error403.html.twig (hunk 1 above)
Comment (lchrusciel) [inline app/Resources/TwigBundle/views/Exception/error403.html.twig:1]: This template could be removed. It will not be used anywhere

File: src/Sylius/Bundle/UiBundle/Resources/views/error.html.twig [hunk 4]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+
+<html lang="{{ app.request.locale|slice(0, 2) }}">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>{{ sylius.channel.name }}</title>
```
Comment (lchrusciel) [inline src/Sylius/Bundle/UiBundle/Resources/views/error.html.twig:8]:
```suggestion
    <title>{{ sylius.channel.name|default('sylius.ui.error'|trans) }}</title>
```

**Comment by jakubtobiasz** [comment]
From what I know from @kulczy this PR is not relevant anymore.
